tabbycat/results/result.py

```diff
@@ -680,6 +698,23 @@ def identical(self, other):
             return False
         return True
 
+    def merge_speaker_order(self, result: BaseDebateResult) -> list[ResultError]:
+        errors = []
+        for side, pos in product(self.sides, self.positions):
+            cur_speaker = self.get_speaker(side, pos)
+            if cur_speaker is None:
+                self.set_speaker(side, pos, result.get_speaker(side, pos))
+            elif result.get_speaker(side, pos) != cur_speaker:
+                errors.append(ResultError("Inconsistent speaker order", "speaker", side, pos))
+                continue  # Don't care about setting ghost/score if can't attribute to the correct speaker
+
+            if not self.get_ghost(side, pos):
+                self.set_ghost(side, pos, result.get_ghost(side, pos))
+            elif not result.get_ghost(side, pos):
+                errors.append(ResultError("Inconsistent ghost order", "ghost", side, pos))
```

Comment:
If `self.get_ghost(side, pos) == False` and `result.get_ghost(side, pos) == True`, then this will have effect of setting the ghost to `True`, not logging the mismatch as an error. Is this the intended behaviour?

Reply:
I'm okay with it, as iron policy is mostly set by the tournament, is minor, and is the most often wrongly set, so that's the field that would be easiest to correct.